rel/files/app.config

```diff
@@ -1,9 +1,50 @@
 [
  {kernel, [
-  {logger_level, warning},
+  {logger_level, notice},
+
+  %% Template with custom fields
+  {mongoose_default_log_template,
+   [colored_start, "when=", time, " level=", level,
+    {what, [" what=", what], ""},
+    {reason, [" reason=", reason], ""},
+    {pid, [" pid=", pid], ""}, " at=", mfa, ":", line, colored_end,
+    {user, [" user=", user], ""},
+    {from_jid, [" from_jid=", from_jid], ""},
+    {to_jid, [" to_jid=", to_jid], ""},
+    {stacktrace, [" stacktrace=", stacktrace], ""},
+    {text, [" text=", text], ""}, " ",
+    msg, "\n"]},
```

Comment:
I'd move this whole template into the `mongoose_flatlog_formatter` file. If it is sane, the MIM configurator will simply add flatlog formatter and be happy and they won't even know such a template exists. If it will not suit them, they can override it by adding theirs here, or in the formatter config, or even they could change the `mongoose_flatlog_formatter` code then.
Is this not a good idea to keep the config cleaner?

Reply:
ok

Reply:
Actually that looks much better now, that was a very good idea! 😄